src/OpenSSL/SSL.py

```diff
@@ -139,6 +140,7 @@ class _buffer(object):
 OP_NO_TLSv1 = _lib.SSL_OP_NO_TLSv1
 OP_NO_TLSv1_1 = _lib.SSL_OP_NO_TLSv1_1
 OP_NO_TLSv1_2 = _lib.SSL_OP_NO_TLSv1_2
+OP_NO_TLSv1_3 = _lib.SSL_OP_NO_TLSv1_3
```

Comment:
This symbol isn't going to exist on OpenSSL<1.1.1

Reply:
Cryptography already handles that case for us:

https://github.com/pyca/cryptography/blob/c7681e80a68a97ba56453e7fbb960f0e59f4acad/src/_cffi_src/openssl/ssl.py#L839-L841